### ISSUE TITLE
Error on dataset.json from the trained weights of AutoPET II ?

### ISSUE THREAD

**Issue by arnaudberenbaum** [MIC-DKFZ/nnUNet#1872]
Hello,

I'm trying to finetune the "AutoPET II Challenge" pre-trained model but I found that there might be an error in the dataset.json:

```json
{
    "channel_names": {
        "0": "CT",
        "1": "CT" // I think it should be "PT" ?
    },
    "labels": {
        "background": 0,
        "tumor": 1
    },
    "numTraining": 1014,
    "file_ending": ".nii.gz",
    "name": "AutoPETII_2023",
    "reference": "https://autopet-ii.grand-challenge.org/",
    "release": "release",
    "description": "AutoPETII_2023"
}
```
In the plans.json : 
```json
"normalization_schemes": [
                "CTNormalization",
                "CTNormalization"
],
```

Should it be PTNormalization?

Do you know if it has impacted the training and the performances?

Thanks a lot

Edit : Based on documentation, maybe it is the only way to perform "CT" type of normalization on PT images ?  Using tumor segmentation as "foreground", does it means that only the tumor portion is normalized ?
(I can move this thread to "discussion")
